### ISSUE TITLE
Support multiple superuser addresses in Ownable

### ISSUE THREAD

**Issue by stefann-01** [gnolang/gno#3347]
Extend `Ownable` to supports two superuser addresses (main and backup). This would allow users to recover contracts if the main address is lost while keeping the simplicity of the existing design for those who don’t need this feature. The `Authorizable` extension allows multiple addresses but still relies on a single superuser, which doesn’t fully address this issue.  

I’m interested in working on this but would first appreciate hearing some thoughts.

**Comment by n2p5** [comment]
Yes. I ran into a similar issue, and it is why I built [mgroup](https://github.com/gnolang/gno/blob/master/examples/gno.land/p/n2p5/mgroup/mgroup.gno) for my [config realm ](https://gno.land/r/n2p5/config). 

I'd love to collaborate on this as well. I made some design choices for `mgroup` (short for Managed Group) that work for what I needed, but I think it could be generalized. What I wanted was:

- Primary Owner - Can and Add backup Owners and members
- Backup Owners (Backup Owners can `ClaimOwnership()` to become the Primary Owner
- Members are like what's in Authorizable, but it has a few features like paginated listing of members, etc.

**Comment by stefann-01** [comment]
Great, your mgroup covers the problem I had in mind. The idea of adding a pagination option is also quite interesting. I think it would be great to have these options available directly in Ownable as extensions to keep it all in one place.

Maybe we could create two extensions: one for adding backup owners and another for enabling both backup owners and authorized members. A useful addition might be the ability to define different roles or levels of authorization for members.

I’d also love to collaborate on this. How do you think we should approach it?
